src/helpers/formatters.js

```diff
@@ -12,11 +12,16 @@ export default {
      * @param value
      * @param args
      */
-    currency: (value, ...args) => {
-        let currencyFormatter = global.currency;
+    formatCurrency: (value, ...args) => {
+        let currencyFormatter = global.OSREC && global.OSREC.CurrencyFormatter;
+        let handlebars = global.Handlebars;
 
-        if (!currencyFormatter) {
-            currencyFormatter = require('currency-formatter');
+        if (isUndefined(currencyFormatter)) {
+            currencyFormatter = require('currencyformatter.js');
+        }
+
+        if (isUndefined(handlebars)) {
```

Comment:
We have this check here too: 
https://github.com/leapfrogtechnology/just-handlebars-helpers/blob/master/src/H.js#L18-L22

Will anyone try to call `formatCurrency` before registering the helpers?